dbt/profiles.yml

```diff
@@ -25,8 +25,4 @@ dbt-civis:
       method: "{{ env_var('GCP_AUTH_METHOD') }}"
       project: "{{ env_var('GCP_PROJECT_ID') }}"
       schema: "{{ env_var('DBT_SCHEMA') }}"
-      refresh_token: "{{ env_var('DBT_ENV_SECRET_DATABASE_PASSWORD') }}"
-      client_id: "{{ env_var('GCP_CLIENT_ID') }}"
-      client_secret: "{{ env_var('GCP_CLIENT_SECRET') }}"
-      token_uri: "{{ env_var('GCP_TOKEN_CREDENTIAL_URI') }}"
-      scopes: "{{ env_var('GCP_SCOPE').split(' ') }}"
+      token: "{{ env_var('DBT_ENV_SECRET_GCP_ACCESS_TOKEN') }}"
```

Comment:
do I need to update anywhere else? I haven't added to this repo yet

Reply:
no this should be all that you need to do.
but may i suggest using `DBT_ENV_SECRET_DATABASE_PASSWORD`
using 1 env variable for all things "password" related would make it easier on the end user.

Reply:
people will most definately have to read our documentation to figure out how to do all of these things. i feel like having good documentation in hand with good clear examples should clear any confusion.

The other option would be to rename `DBT_ENV_SECRET_DATABASE_PASSWORD` to something more broad like 
* `DBT_ENV_DATABASE_CONNECTION_SECRET`
* `DBT_ENV_SECRET_DATABASE_CREDENTIAL`

but that's not a hill i'm willing to die on.

Reply:
yeah that's fair. Im hesitant to change the other vars at this time, but maybe we can talk about it as a team on monday? It could be nice to have a generic single var name. But for now, Im liking `DBT_ENV_SECRET_GCP_ACCESS_TOKEN`